### ISSUE TITLE
[WDS] Nothing changed.

### ISSUE THREAD

**Issue by aep** [webpack/webpack-dev-server#850]
```
const path = require('path');
module.exports = {
    devServer: {
        contentBase: path.join(__dirname, "dist")
    },
    entry: './src/app.js',
    output: {
        path:       path.join(__dirname, 'dist'),
        filename:   'bundle.js'
    }
};
```

app.js is empty.
When i now add random garbage to app.js (be it correct syntax or not), dev-server correctly spots the change and starts compiling, but apparantly not the file i just saved.
```
webpack: Compiling...
Hash: 5a194bf652a9d1f48771
Version: webpack 2.2.1
Time: 33ms
chunk    {0} bundle.js (main) 300 kB [entry]
   [36] ./src/app.js 60 bytes {0} [built]
     + 85 hidden modules
webpack: Compiled successfully.
```

the hash does not change, no matter what i put in app.js

and chrome dev console says:
[WDS] Nothing changed.

and manually GETing /bundle.js shows its still serving the old file.

However, when i just restart the server, the hash changes and the new content is used.

**Comment by SpaceK33z** [comment]
This is a usage error, please ask on stackoverflow, you'll get better help there.

**Comment by theBull** [comment]
@SpaceK33z if you know it's a usage error why not give a hint of what the usage error is...?

**Comment by shellscape** [comment]
@theBull because most folks will benefit from learning how something works versus being fed answers. given the age of this issue, I'd also wager that keez can tell it's user error based on given input and output, but can't say specifically what it is given the limitation of info in the issue. Hashes are also an issue with `webpack` proper, rather than `webpack-dev-server`.

I'll add that it appears you stopped by just to drop a comment that had no real purpose other than to try to paint keez as a jerk. We don't welcome that here.

**Comment by theBull** [comment]
Not trying to paint anyone as a jerk, and I'm not just stopping by to drop a comment without a purpose. I'm trying to use the `webpack-dev-server` and have ran into a similar issue practically out-of-the-box, which also appears to be pretty widely met by others. After spending a thorough amount of time crawling through various related issues, as well as scouring SO for related issues, the consensus I gathered was both that keez seems to understand the system quite well, and others (like me) would benefit from even the tiniest of nuggets indicating what the user error might be. Closing an issue with some knowledge of what's causing the issue, without sharing that knowledge isn't very helpful. I don't fault you for stepping in to defend someone, but if I see something I'm going to say something. At the end of the day, I'm not the only one who's likely going to stumble on this issue and find a dead end, so it'd be beneficial for the community as a whole to at least get *something* helpful from those with the knowledge aside from the obligatory redirect to SO.
